src/pipecat/services/ai_services.py

```diff
@@ -419,7 +423,7 @@ async def _update_settings(self, settings: Dict[str, Any]):
                 logger.debug(f"Updating STT setting {key} to: [{value}]")
                 self._settings[key] = value
                 if key == "language":
-                    self._settings[key] = Language(value)
+                    await self.set_language(value)
```

Comment:
Do we have more STT services that require something special?

Reply:
Not that I'm aware of.